plugins/jira-dashboard-backend/src/service/router.ts

```diff
@@ -85,18 +82,11 @@ export async function createRouter(
   });
 
   router.get(
-    '/dashboards/by-entity-ref/:kind/:namespace/:name',
+    '/dashboards/by-entity-ref/:entityRef',
```

Comment:
Please keep the ":kind/:namespace/:name" parameters. See issue #76 for the history.

Reply:
Updated